charts/rode-ui/values.yaml

```diff
@@ -8,6 +8,32 @@ rode:
   # App default is http://localhost:50052
   url: http://rode:50052
 
+  ## Rode authentication settings
+  ## Use rode.auth.oidc.enabled to enable OIDC auth
+  ##
+  auth:
+    oidc:
+      enabled: false
+      clientId: ""
+      clientSecret: ""
+      issuerUrl: ""
+
+    ## appSecret is the encryption key that is used to encrypt session cookies. this will be auto-generated if it isn't set
+    ##
+    appSecret: ""
+
+## appUrl represents the URL users can use to reach the application in their browser. this is used when redirecting users after signing in via SSO
+##
+appUrl: http://localhost:3000
+
+## extraEnvVars is a list of extra environment variables to be set on the rode-ui container
+## Example:
+## extraEnvVars:
+##   - name: NODE_TLS_REJECT_UNAUTHORIZED
+##     value: "0"
+##
+extraEnvVars: []
```

Comment:
Typically this in other charts as `extraEnv`

Reply:
all of the the [bitnami charts](https://github.com/bitnami/charts) uses `extraEnvVars` which is why I decided to do this.  I don't feel strongly one way or the other though, so I can change it if you want.

Reply:
I'm fine leaving it as is. The [Rode](https://github.com/rode/charts/blob/650bc28ab55e0daec60a6b8406d8c8b998e02902/charts/rode/templates/deployment.yaml#L52-L55) and [build collector](https://github.com/rode/charts/blob/3926f2f400e6ecf227eba64af5f57a99934d060f/charts/rode-collector-build/templates/deployment.yaml#L47-L50) use `extraEnv`, but I think at some point we'll want to make a pass to align all of the charts anyway.

Reply:
I definitely agree there.  I've been looking through some of the bitnami charts for inspiration and they have some really solid practices nailed down that I think we could borrow.